<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>label_line_editClass</class>
 <widget class="QWidget" name="label_line_editClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>label_line_edit</string>
  </property>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>image</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>101</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>show input</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="label_line_edit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn</sender>
   <signal>clicked()</signal>
   <receiver>label_line_editClass</receiver>
   <slot>slot_show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_show()</slot>
 </slots>
</ui>
